modules/cnv-edit-boot-order-web.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+
+[id="cnv-edit-boot-order-web_{context}"]
+= Editing a boot order list in the web console
+
+Edit the boot order list in the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
+. Click the pencil icon for *Boot Order*.
```

Comment:
Can you be more specific about where the pencil icon is in relation to `*Boot Order*`? Like "on the right side" or something like that?

Reply:
How about:
Click the pencil icon next to the Boot Order.

Reply:
Please change this to match the other module (`Click the pencil icon that is located on the right side of *Boot Order*.`)